bodhi/server/services/graphql.py

```diff
@@ -17,18 +17,56 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Defines API endpoints related to GraphQL objects."""
 from cornice import Service
+from webob_graphql import serve_graphql_request
+import graphene
+
+from bodhi.server.graphql_schemas import Release
+
 
 graphql = Service(name='graphql', path='/graphql', description='graphql service')
 
 
 @graphql.get()
 def graphql_get(request):
     """
-    Return "Hello World".
+    Perform a GET request.
+
+    Args:
+        request (pyramid.Request): The current request.
+    Returns:
+        The response from Bodhi to the request.
+    """
+    context = {'session': request.session}
+
+    # Setting graphiql_enabled=True
+    return serve_graphql_request(request, schema, graphiql_enabled=True, context_value=context)
+
+
+@graphql.post()
+def graphql_post(request):
```

Comment:
I am not 100% sure here (@StephenCoady might look at that), but I am not sure that we need 2 function here we could have only one with the following declaration

```
@graphql.post()
@graphql.get()
def graphql_endpoint(request):
....
````

Reply:
This is correct, function with both `post` and `get` will work the same